Changelog

```diff
@@ -20,25 +20,62 @@ NiBabel
 +++++++
 
 Most work on NiBabel so far has been by Matthew Brett (MB), Michael Hanke (MH)
-and Stephan Gerhard (SG).
+Ben Cipollini (BC), Marc-Alexandre Côté (MC), Chris Markiewicz (CM), Stephan
+Gerhard (SG) and Eric Larson (EL).
 
 References like "pr/298" refer to github pull request numbers.
 
-* Upcoming
+* 2.1 (TBA)
+
+  We dropped support for Pythons 2.6 and 3.2 for this release.
 
-  * Read and write support for DICOM tags in NIfTI Extended Header using
-    pydicom (pr/296);
-  * Trackvis reader will now allow final streamline to have fewer points that
-    tne numbe declared in the header, with ``strict=False`` argument to
-    ``read`` function;
-  * Minor API breakage in trackvis reader. We are now raising a DataError if
-    there are too few streamlines in the file, instead of a HeaderError.  We
-    are raising a DataError if the track is truncated when ``strict=True``
-    (the default), rather than a TypeError when trying to create the points
-    array.
   * New API for managing streamlines and their different file formats. This
     adds a new module ``nibabel.streamlines`` that will eventually deprecate
-    the current trackvis reader found in ``nibabel.trackvis``.
+    the current trackvis reader found in ``nibabel.trackvis`` (pr/391) (MC,
+    reviewed by Bago Amirbekian, Samuel St-Jean, MB);
```

Comment:
Really small nitpick: I don't know what is the threshold of implication to be mentionned here, but I did quite a bit of review on PR 391. If it's worth a mention, feel free to add me :)